src/chrome/content/rules/Surge.sh.xml

```diff
@@ -11,5 +11,6 @@
 
 	<rule from="^http:"
 		to="https:" />
+  <exclusion pattern="http://test-http\.surge\.sh/" />
```

Comment:
```suggestion
  <exclusion pattern="^http://test-http\.surge\.sh/$" />
```

Please move the exclusion above the rule and add a comment documenting the reason.

Reply:
Done!

Reply:
Ah sorry, every exclusion needs an explicit test url.